content/en/docs/reference/glossary/sig.md

```diff
@@ -2,7 +2,7 @@
 title: SIG (special interest group)
 id: sig
 date: 2018-04-12
-full_link: https://github.com/kubernetes/community/blob/master/sig-list.md#master-sig-list
+full_link: https://github.com/kubernetes/community/blob/master/sig-list.md#special-interest-groups
```

Comment:
I think it would be better if we refer them to the https://k8s.dev version.

```suggestion
full_link: https://www.kubernetes.dev/community/community-groups/#special-interest-groups
```

Reply:
Thank you for your review @Sea-n, I didn't knew that there was https://k8s.dev site!
fixed it again! 😀